src/yunohost/log.py

```diff
@@ -192,7 +187,7 @@ def log_display(path, number=50, share=False):
                 if 'log_path' in metadata:
                     log_path = metadata['log_path']
             except yaml.YAMLError:
-                error = m18n.n('log_corrupted_md_file', file=md_path)
+                error = m18n.n('log_corrupted_md_file', md_file=md_path)
```

Comment:
Well it would be nice to just use `read_yaml` here too ...

But I can't understand what's the intention in putting the `info['medata']` things inside the `try` block, nor what's the intention with displaying just a warning or raising an exception depending if `log_path` actually exists ... Can't we factorize out the loading of the yaml ?